src/Elliptic/Systems/Xcts/Events/ObserveAdmIntegrals.cpp

```diff
@@ -75,32 +94,41 @@ void local_adm_integrals(
                                       face_extrinsic_curvature(ti::k, ti::l));
 
     // Compute curved area element
-    const auto face_sqrt_det_spatial_metric =
-        Scalar<DataVector>(sqrt(get(determinant(face_spatial_metric))));
+    const auto face_sqrt_det_conformal_metric =
+        Scalar<DataVector>(sqrt(get(determinant(face_conformal_metric))));
     const auto curved_area_element =
         area_element(face_inv_jacobian, boundary_direction,
-                     face_inv_spatial_metric, face_sqrt_det_spatial_metric);
+                     face_inv_conformal_metric, face_sqrt_det_conformal_metric);
 
     // Get face mesh and face normal from data box
     const auto& face_mesh = mesh.slice_away(boundary_direction.dimension());
-    // Note that this face normal is related to the conformal metric. To get
-    // the non-conformal face normal, we multiply by the square of the
-    // conformal factor in the computation of 'contracted_integrand'.
-    const auto& face_normal_conformal =
+    const auto& conformal_face_normal =
         conformal_face_normals.at(boundary_direction);
 
-    // Compute surface integrand
-    const auto surface_integrand = Xcts::adm_linear_momentum_surface_integrand(
-        face_conformal_factor, face_inv_spatial_metric,
-        face_inv_extrinsic_curvature, face_trace_extrinsic_curvature);
-    const auto contracted_integrand = tenex::evaluate<ti::I>(
-        surface_integrand(ti::I, ti::J) * square(face_conformal_factor()) *
-        face_normal_conformal(ti::j));
+    // Compute surface integrands
+    const auto mass_integrand = Xcts::adm_mass_surface_integrand(
+        face_deriv_conformal_factor, face_inv_conformal_metric,
+        face_conformal_christoffel_second_kind);
+    const auto linear_momentum_integrand =
+        Xcts::adm_linear_momentum_surface_integrand(
+            face_conformal_factor, face_inv_spatial_metric,
+            face_inv_extrinsic_curvature, face_trace_extrinsic_curvature);
+
+    // Contract surface integrands with face normal
+    const auto contracted_mass_integrand =
+        tenex::evaluate(mass_integrand(ti::I) * conformal_face_normal(ti::i));
+    const auto contracted_linear_momentum_integrand = tenex::evaluate<ti::I>(
+        linear_momentum_integrand(ti::I, ti::J) *
+        square(face_conformal_factor()) * conformal_face_normal(ti::j));
```

Comment:
Something is probably wrong here with the powers of the conformal factor. You changed the area element to use the conformal metric (which I think is good), but this means you also have to adjust the powers of the conformal factor here. We can merge it like this and then fix it for general conformal factors, but then please add a comment here or in the docs of the class that it's only tested for psi=1 so far.